index.html

```diff
@@ -85,6 +108,18 @@ <h2>
       Cases</a>
       </li>
     </ul>
+    <h3>
+      Multiple User Agents
```

Comment:
I'd encourage avoiding using the term User Agents for wallets ... it is a fairly overloaded term in Web Platform, and from a security threat modeling perspective, the Wallet has a very different position than the Browser, so I'd avoid using the same term to describe both.